### PR TITLE
Hover reactions bar

### PR DESCRIPTION
**Pull request by joaojmendes**
| Q               | A
| --------------- | ---
 
 
| New sample?      | [ x]
 

#### What's in this Pull Request?

 New Control HoverReactionsBar

### CODE REVIEW

File: src/controls/HoverReactionsBar/components/reactionPicker/ReactionPicker.tsx [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+
+import emojiList from '../../data/fluentEmojis.json';
+import { useFluentEmojis } from '../../hooks/useFluentEmojis';
+import { useOnClickOutside } from '../../hooks/useOnClickOutside';
+import { IEmojiInfo } from '../../models/IFluentEmoji';
+import { ReactionGroups } from './ReactionGroups';
+import { RenderEmoji } from './RenderEmoji';
+import { useReactionPickerStyles } from './useReactionPickerStyle';
+
+const PICKER_WIDTH = 350; //  450 height of picker + 50 from overtoolbar
```
Comment (joelfmrodrigues) [inline src/controls/HoverReactionsBar/components/reactionPicker/ReactionPicker.tsx:20]: Are these comments relevant or correct?

File: src/controls/HoverReactionsBar/components/reactionPicker/ReactionPicker.tsx [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+    },
+    [emojiList, onSelect, onDismiss, groups, returnType]
+  );
+
+  React.useEffect(() => {
+
+    const countainerBounds = target?.getBoundingClientRect() as DOMRect;
+    let pickerTopPosition = 0;
+    let pickerLeftPosition = 0;
+    // test if list of mentiosn can be displayed in full
```
Comment (joelfmrodrigues) [inline src/controls/HoverReactionsBar/components/reactionPicker/ReactionPicker.tsx:94]: Is this comment still relevant?

File: src/controls/HoverReactionsBar/components/reactionPicker/ReactionPicker.tsx [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+    [onSelect, onDismiss, groups, returnType]
+  );
+
+  return (
+    <>
+      <Card className={styles.card} style={pickerStyles} ref={pickerRef}>
+        <div className={styles.cardContent}>
+          <Input
+            value={searchValue}
+            placeholder="Search Emoji"
```
Comment (joelfmrodrigues) [inline src/controls/HoverReactionsBar/components/reactionPicker/ReactionPicker.tsx:172]: Add placeholder string to loc files please

File: src/webparts/controlsTest/components/TestControl.tsx
Comment (joelfmrodrigues) [inline src/webparts/controlsTest/components/TestControl.tsx:1]: Any chance you could add both TermSetNavigation and also HoverReactionsBar to the main test web part component (ControlsTest.tsx)?

**Review by joelfmrodrigues — CHANGES_REQUESTED** [review]
@joaojmendes just some small notes :)

File: src/controls/HoverReactionsBar/components/reactionPicker/ReactionPicker.tsx (hunk 1 above)
Comment (joaojmendes) [reply src/controls/HoverReactionsBar/components/reactionPicker/ReactionPicker.tsx:20]: fixed

File: src/controls/HoverReactionsBar/components/reactionPicker/ReactionPicker.tsx (hunk 2 above)
Comment (joaojmendes) [reply src/controls/HoverReactionsBar/components/reactionPicker/ReactionPicker.tsx:94]: fixed

File: src/controls/HoverReactionsBar/components/reactionPicker/ReactionPicker.tsx (hunk 3 above)
Comment (joaojmendes) [reply src/controls/HoverReactionsBar/components/reactionPicker/ReactionPicker.tsx:172]: fixed

**Pull request merged**